utils/installer/uninstall.sh

```diff
@@ -24,7 +24,7 @@ function usage() {
   echo ""
   echo "Options:"
   echo "    -h, --help                       Print this help message"
-  echo "    --remove-config                  Remove old backup folders as well"
+  echo "    --remove-config                  Remove user config files  as well"
```

Comment:
```suggestion
  echo "    --remove-config                  Remove user config files as well"
```

Reply:
that double space was accidental

but i left there because 'as well' in both lines aligned correctly😅
![image](https://user-images.githubusercontent.com/68694876/203347248-83ddf51c-3bbd-40e4-aecf-dd3d62fbea7e.png)

but better to remove

Reply:
```
  echo "    --remove-config                  Remove user config files as well"
  echo "    --remove-backups                 Remove old backup folders as well"
  echo ""

  echo "    --remove-config                  Remove user config folder as well"
  echo "    --remove-backups                 Remove old backup folders as well"

  echo "    --remove-config                  Remove user config files  as well"
  echo "    --remove-backups                 Remove old backup folders as well"

```

folder would have been better.. :)

Reply:
doesn't matter it's just the uninstaller, it doesn't have to align
it's merged, don't worry about it, thanks for the pr :)